bower.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "gpii-binder",
+  "version": "0.0.0",
```

Comment:
May as well just start at 1.0.0 so this can be published and retrieved properly from npm